apps/scripts/Bubble.c

```diff
@@ -0,0 +1,43 @@
+void swap(int *xp, int *yp)
```

Comment:
Add header to the file.
- short description.
- Who is the owner of this test.
- when was this test created 
comment and Explain the memory regions in this test